tests/streams/stringifier-stream.js

```diff
@@ -11,6 +11,31 @@ function getTemplate(template, specialTags, pipeBeforeTags) {
 }
 
 describe('Stringifier Stream', () => {
+    let index = 0;
+    const delays = [30, 20, 50];
+    function getDelay() {
+        const currDelay = delays[index];
+        index++;
+        if (index >= delays.length) {
+            index = 0;
+        }
+        return currDelay;
```

Comment:
the test should not rely on the timing (and it does not seem to do so), so why not actual pseudo random values?